frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -1,17 +1,43 @@
 <script setup lang="ts">
+import { computed } from 'vue';
 import LabelText from '@/components/design-system/typography/LabelText.vue';
 
 export interface Props {
   active?: boolean;
 }
 
-withDefaults(defineProps<Props>(), {
+const classes = computed(() => ({
+  'pill--active': props.active,
```

Comment:
Pojďme to využít scoped CSS: 

```suggestion
  active: props.active,
```